lighthouse-extension/app/src/popup.js

```diff
@@ -37,6 +37,32 @@ document.addEventListener('DOMContentLoaded', _ => {
   const optionsList = document.body.querySelector('.options__list');
   const okButton = document.getElementById('ok');
 
+  function getLightHouseVersion() {
+    return chrome.runtime.getManifest().version;
+  }
+
+  function getChromeVersion() {
+    return /Chrome\/([0-9.]+)/.exec(navigator.userAgent)[1];
+  }
+
+  function buildReportErrorLink(err) {
+    const reportErrorEl = document.createElement('a');
+    reportErrorEl.className = 'button button--report-error';
```

Comment:
yeah, my point was more styling. It's a link acting like a link, but somewhat styled like a button :)

However, it works and we've talked about reworking the popup as it keeps getting more crowded anyways, so we can address this at that point if need be. I'd like to get this landed now to help users out.